portfolio/src/main/webapp/style.css

```diff
@@ -218,4 +220,126 @@ html, body {
 
 #accordion {
     margin-bottom: 20px;
+}
+
+/* COMMENTS */
```

Comment:
For FE changes, I like to attach before and after screenshots to the PR descriptions. It's a quick and easy visual way to know what you did without reading code.

Reply:
Right, I keep forgetting to do that! I just added a screenshot to this PR and will make sure to add them for future PRs!